src/commands/utility/pr-template.js

```diff
@@ -1,21 +1,23 @@
-const { SlashCommandBuilder, ChannelType } = require('discord.js');
+const { SlashCommandBuilder } = require('discord.js');
 const { createPRModal } = require('../../modals/pr-template-modal');
+const { PR_TEMPLATE_ALLOWED_CHANNELS } = require('../../config');
 
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('pr-template')
-    .setDescription('Create a formatted PR review request')
-    .addChannelOption((option) =>
+    .setDescription('Send a formatted PR review request')
+    .addStringOption((option) =>
       option
         .setName('channel')
-        .setDescription('Select channel to send the PR review request')
+        .setDescription('Select a channel for the PR review request')
         .setRequired(true)
-        .addChannelTypes(ChannelType.GuildText)
+        .addChoices(...PR_TEMPLATE_ALLOWED_CHANNELS)
```

Comment:
addChoices doesn't accepts an array directly?

```suggestion
        .addChoices(PR_TEMPLATE_ALLOWED_CHANNELS)
```

Reply:
Yes, it works